### PR TITLE
Add doc for chained SQL queries.

### PR DESCRIPTION
**Pull request by likawind**
This PR adds documentation for chained SQL queries. We put this in a standalone section so that it's easily to find from index.

Preview in VS Code:
Screen Shot 2022-10-27 at 10 40 36 AM

### CODE REVIEW

File: integrations/using-integrations/relational-databases.md [hunk 1]
```diff
@@ -30,6 +30,23 @@ Every SQL database has a slightly different dialect of SQL, and Aqueduct does no
 In order to validate that your query works correctly, call `.get()` on the artifact Aqueduct returns to you. If there's an error, you will be notified immediately.
 {% endhint %}
 
+### Reading Using a Chain of Queries
```
Comment (vsreekanti) [inline integrations/using-integrations/relational-databases.md:33]: Nit: "Chaining SQL Queries Together"

File: integrations/using-integrations/relational-databases.md [hunk 2]
```diff
@@ -30,6 +30,23 @@ Every SQL database has a slightly different dialect of SQL, and Aqueduct does no
...
+Aqueduct supports extracting from a database using a chain of queries. This allows you to break down complex queries into smaller pieces for better readability.
+
+To do so, simply call `.sql()` with a list of strings. In the query chain, you can use the special placeholder `$` refering to the 'previous output' in the chain. The output of the last query will be returned as the output artifact of this execution.
```
Comment (vsreekanti) [inline integrations/using-integrations/relational-databases.md:36]: Nit: remove quotes around previous output.

**Review by vsreekanti — APPROVED** [review]
Mostly looks good, @likawind! Just a couple edits inline.

**Comment by likawind** [comment]
@vsreekanti updated, see screenshot in PR description

File: integrations/using-integrations/relational-databases.md [hunk 3]
```diff
@@ -30,6 +30,30 @@ Every SQL database has a slightly different dialect of SQL, and Aqueduct does no
...
+
+```python
+customers = demo_db.sql([
+  'SELECT * FROM customers WHERE company_size < 50;',
+  'SELECT * FROM $ WHERE n_workflows < 10;', # `$` represents the output of the above query.
+  'SELECT cust_id AS id, n_data_eng FROM $;', # The output of this query will be returned.
+])
+```
+
+This example returns the `cust_id` and `n_data_eng` of all customers with `company_size < 50` and `n_workflows < 10`. Underhood, we compile this chain to a single query using a `WITH` clause, and the compiled query is executed:
```
Comment (vsreekanti) [inline integrations/using-integrations/relational-databases.md:48]: Looks good! Minor typo: Underhood -> Under the hood

**Review by vsreekanti — APPROVED** [review]

**Pull request merged**